php-config/Slate/People/Student.config.d/current-sections-relationship.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Slate\People;
+
+use DB;
+
+use Slate\Courses\Section;
+use Slate\Courses\SectionParticipant;
+use Slate\Term;
+
+$courseSectionsInTerm = DB::allValues(
+    'ID',
+    '
+        SELECT ID
+          FROM `%s`
+         WHERE TermID IN (%s)
+    ',
+    [
+        Section::$tableName,
+        join(',', Term::getClosest()->getMaster()->getContainedTermIDs())
+    ]
+);
+
+Student::$relationships['CurrentSections'] = [
+    'type' => 'many-many'
+    ,'class' => Section::class
+    ,'linkClass' => SectionParticipant::class
+    ,'linkLocal' => 'PersonID'
+    ,'linkForeign' => 'CourseSectionID'
+    ,'conditions' => [
+        sprintf('(Link.CourseSectionID IN(%s))', join(',', $courseSectionsInTerm)),
+        '(Link.StartDate IS NULL OR DATE(Link.StartDate) <= CURRENT_DATE)',
+        '(Link.EndDate IS NULL OR DATE(Link.EndDate) >= CURRENT_DATE)'
+    ]
```

Comment:
`'many-many'` does support callable `'conditions'`: https://github.com/JarvusInnovations/emergence-skeleton/blob/a05f5fed622fd850ea840efff952d4f1eb9bfa61/php-classes/ActiveRecord.class.php#L2552

So this relationship can go right under the `Student` class and do something like:

```suggestions
    'conditions' => [__CLASS__, 'getCurrentSectionsConditions']
```